edge-modules/ModuleLib/ModuleUtil.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace ModuleLib
```

Comment:
We should follow the namespace convention. Microsoft.Azure.Devices.Edge.ModuleUtil